src/package/store.rs

```diff
@@ -260,6 +275,13 @@ impl PackageStore {
     }
 }
 
+pub struct Entry {
+    /// Actual bytes of the file
+    pub contents: Bytes,
+    /// last modified time, as number of seconds since EPOCH, if available
+    pub mtime: Option<u64>,
```

Comment:
Can we eventually just store the whole metadata structure here? https://doc.rust-lang.org/nightly/std/fs/struct.Metadata.html